starter_code/elevator.js

```diff
@@ -5,15 +5,25 @@ class Elevator {
     this.requests   = [];
   }
 
-  start() { }
-  stop() { }
-  update() { }
+  start() {
+    const startElevator = setInterval(function(){
```

Comment:
You may be loosing context in here. Better use arrow functions
```
 this.interval = setInterval( () => { this.update() }, 1000 );
```
startElevator should be defined in the constructor, and it should have a better name like interval